### PR TITLE
V116 NETDATA 

### PR DESCRIPTION
**Pull request by rhkean**
implements netdata (realtime perfomance monitoring) to dietpi (issue #267 )

will need to correct the download link when moved out of testing folder.
currently, added to all menus, but still need to test on other boards (only tested on RPi 2)

### CODE REVIEW

File: dietpi/dietpi-services [hunk 1]
```diff
@@ -68,6 +69,8 @@
 			elif [ -f /etc/init.d/"${aSERVICE_NAME[$i]}" ]; then
 				aSERVICE_AVAILABLE[$i]=1
 
+			elif [ -f /etc/systemd/system/"${aSERVICE_NAME[$i]}".service ]; then
```
Comment (Fourdee) [inline dietpi/dietpi-services:72]: Nice one, support for systemd :+1:

File: dietpi/dietpi-software [hunk 2]
```diff
@@ -462,6 +464,7 @@ _EOF_
 		EMONHUB=$(cat "$FP_INSTALLED_FILE" | grep -m1 'EMONHUB ' | awk '{print $2}')
 		EMONCMS=$(cat "$FP_INSTALLED_FILE" | grep -m1 'EMONCMS ' | awk '{print $2}')
 		RPIMONITOR=$(cat "$FP_INSTALLED_FILE" | grep -m1 'RPIMONITOR ' | awk '{print $2}')
+		NETDATA=$(sed -n 's/^NETDATA\s*\(.*\)/\1/p' "$FP_INSTALLED_FILE")
```
Comment (Fourdee) [inline dietpi/dietpi-software]: Need to add a space after `s/^NETDATA`. This prevents items with NETDATA at the start from being picked up in the file.

EG:
NETDATA
NETDATA_SOMEADDITION
Will both be picked up without the space at the end.

**Comment by Fourdee** [comment]
@rhkean 
Excellent work on this Rob :+1: . I've added some comments for recommended changes, if you could go through them and update the code as needed, I'd be grateful.

Once thats done, i'll merge the changes and test the new install on my C2 and RPi v1 (armv6 test). We will need to remove the VM installation option, unless we have a x64.tgz file available for it?

EDIT:
I've moved the DL to http://dietpi.com/downloads/binaries/all/netdata_1.0.0_armhf.tgz as per your request, if you can update the link in sourcecode, i'd be greatful.

EDIT2:
I dont mind doing the changes if you prefer. But i'd like you to have full credit for your work.

File: dietpi/dietpi-software (hunk 2 above)
Comment (rhkean) [inline dietpi/dietpi-software]: yeah... that's what the "\s*" was supposed to do... but it should be "\s+" 
\s:  whitespace
*:  0 or more occurances
+: 1 or more occurances

I'll fix it

I favored the sed call over the cat|grep|awk for minimal process spawning, but if you'd prefer I maintain consistency, I can change it.  (thoughts?)

**Comment by rhkean** [comment]
to confirm that I'm not missing something:
- correct the sed regular expression
- update the URL
- remove the VMx64 menu option (or create the binary)

correct?

EDIT:
I've made these changes.
I've also never done a pull request with changes.  Do I just close this one and issue another pull request?  Or is there a different process?  thnx

File: dietpi/dietpi-services (hunk 1 above)
Comment (rhkean) [inline dietpi/dietpi-services:72]: yeah... this probably means that it won't work for on wheezy, but we really are supporting this except for legacy.  right?

Comment (Fourdee) [inline dietpi/dietpi-services:72]: Yep, wheezy is being phased out (as per official raspbian image). We are not adding additional software for it either, so best to remove the installation option for wheezy.

**Comment by Fourdee** [comment]
Me neither Lol. Probably best to create a new pull request, and link the new request url in a comment reply here, so we can follow it.

**Comment by Fourdee** [comment]
Yep, changes as per your list / reply 
:+1:

**Comment by rhkean** [comment]
I made that comment this morning.  It looks like (according to the "commits" tabs above) github recognizes the new commits.

we can take the opportunity to test and see... :)

I'll make 1 last inconsequential change and see if it gets pulled it with the rest... if not, I'll create a 2nd pull request.

EDIT:
yup, I added a comment to the patch file.... it's showing up in the changes

**Comment by rhkean** [comment]
testing locally (created new rhkean branch)
tested full upgrade process on Pi2 from v111 image to v116:  success
currently running same test on a Pi1 B+

EDIT:
Pi1 B+ ---- FAILED
I'll build a new package for it tomorrow

**Comment by Fourdee** [comment]
@rhkean 
Excellent, I learnt something new today and its not even lunchtime yet lol :+1:  
I'll merge this now so I can test Odroids. We can add the RPi v1 (armv6) link at a later date, with a new pull request if needed.

**Comment by Fourdee** [comment]
Made a ticket to continue from this pull request: https://github.com/Fourdee/DietPi/issues/276

**Pull request merged**